.github/workflows/main.yml

```diff
@@ -15,7 +15,9 @@ jobs:
           python-version: "3.8"
           cache: "pip"
           cache-dependency-path: requirements.*.txt
```

Comment:
`setup-action` also wraps `setup-python` so you can remove the bit before this too

Reply:
Ah, yes!

For some reason, I thought there'd been some hiccup around that, and so didn't add it, but I can't recall or find what that was.

Reply:
I think you might be thinking of `checkout`? IIRC we tried putting it in and it caused issues between the action and the repo code checkouts.

Reply:
Thanks; yes, it might have been that.